tasks/distro.js

```diff
@@ -22,10 +22,15 @@ const {
   nightly,
   publish,
   config,
-  'on-demand': onDemand,
-  region
+  'on-demand': onDemand
```

Comment:
You're sure the region has to be set for `onDemand` only? Recent versions of AWS_SDK require region to always be set. I've experienced a similar thing when upgrading `wuffle` a while back.

Reply:
I believe that if we use limited credentials for the release job, we will also have to set the region for that job. However, this is not the case yet and I didn't want to touch other topics here.